kerl

```diff
@@ -152,7 +152,9 @@ log_install_cmd() {
 # Check if tput is available for colorize
 tp=$(tput sgr0 1>/dev/null 2>&1 && printf "OK" || printf "")
 if [ -z "$tp" ]; then
-    nocolor "Colorization disabled as 'tput' (via 'ncurses') seems to be unavailable."
+    if [ -z "$CI" ]; then
+        nocolor "Colorization disabled as 'tput' (via 'ncurses') seems to be unavailable."
+    fi
```

Comment:
This happens because we recently opened CI-based KERL_DEBUG-enabled runs to output non-colored text, and in this pull request we set env `KERL_DEBUG=yes`, which means `source $(./kerl path install_"$_KERL_VSN")/activate` would try to execute `Colorization` 😄

Reply:
:see_no_evil: